utils/appliance/__init__.py

```diff
@@ -142,6 +148,36 @@ def uninstall_ipa_client(self):
         assert return_code != 0
 
 
+class ApplianceLoggerAdapter(logging.LoggerAdapter):
```

Comment:
Yes, I was thinking about it. Some sort of StackingTagAddingAdaptor with a class varialbe specifying the tag added.